api-server/src/server/boot/challenge.js

```diff
@@ -406,6 +405,19 @@ export async function modernChallengeCompleted(req, res, next) {
       return next(err);
     }
 
+    if (user.webhook) {
+      // emit webhook here
+      fetch(user.webhook, {
+        method: 'POST',
+        body: JSON.stringify({
+          username: user.username,
+          completedChallenge,
+          points
+        }),
+        headers: { 'Content-Type': 'application/json' }
+      });
```

Comment:
I cannot think of something more useful than just throwing away any error.

```suggestion
      }).catch(() => {});
```